client/source/components/RR/ReviewList/PhotoModal.jsx

```diff
@@ -42,8 +32,8 @@ const Wrapper = styled.div`
   z-index: 1010;
   justify-content: center;
   align-items: flex-start;
-  overflow: auto;
-`;
+  overflow: hidden;
```

Comment:
... but you can still scroll the background! Check out how that is prevented in the shared modal.